### DISCUSSION TITLE
Deployment/hosting recommendations

### DISCUSSION THREAD

**Discussion by nickluger** [blitz-js/blitz#2619] [Q&A] [answered]
Hey Blitz-community!

We're super excited about the ideas and community behind Blitz and thinking about using it for our project instead of Next.js + API. We're hosting on Vercel (frontend) and AWS (API) currently. I'm curious - what is your recommendation for hosting a Blitz app?

- You advise [against](https://blitzjs.com/docs/deploy-vercel) using Vercel for fullstack, but the text reads like you advise against hosting serverless in general, because [Problem 2](https://blitzjs.com/docs/deploy-vercel#idle-connections) is an issue for all serverless platforms. Is Blitz better suited for servers than serverless?
- [Problem 3](https://blitzjs.com/docs/deploy-vercel#database-throughput) (non-idle connection limit), on the other hand, seems like a generic problem of typical relational databases or is there a reason serverless platforms are especially affected by this?
- Is https://render.com/ a good solution?
- Or can we start using [Flightcontrol](https://flightcontrol.dev/?ref=blitzjs) during development and go to production at the end of this year?
- Flightcontrol seems to offer both, containers and serverless. When using serverless, is there a reason it will not be affected by the serverless issues stated above? (Besides cold-starts, which probably everyone has 😄)

Thanks in advance!

**Comment by flybayer** [comment, marked as answer]
The main problem with serverless right now is using relational DBs. Blitz itself will run serverless just fine, same as Next.js.

Most people seem to be using Render currently for hosting Blitz apps.

Will definitely recommend Flightcontrol once we get the MVP launched :D We will be working on the serverless DB issue.

**Reply by nickluger** [reply]
I understand, thanks for the reply 👍
> Will definitely recommend Flightcontrol once we get the MVP launched

Is there already an alpha version of Flightcontrol released for the "prelaunch insiders"?

**Reply by flybayer** [reply]
> Is there already an alpha version of Flightcontrol released for the "prelaunch insiders"?

Not yet. Expect to start onboarding first users in a couple months
